app-tracking-protection/vpn-internal/src/main/java/com/duckduckgo/vpn/internal/feature/VpnInternalSettingsActivity.kt

```diff
@@ -188,9 +189,12 @@ class VpnInternalSettingsActivity : DuckDuckGoActivity() {
                 binding.debugLoggingToggle.isEnabled = isEnabled
                 binding.transparencyModeToggle.isEnabled = isEnabled
                 // only show this toggle when the variant is correct
-                binding.vpnNewNetworkingLayerToggle.isVisible = vpnNetworkStackVariantManager.getVariant() == "ng"
-                binding.vpnNewNetworkingLayerToggle.isEnabled = isEnabled
-                binding.settingsInfo.isVisible = !isEnabled
+                binding.vpnNewNetworkingLayerToggle.isEnabled = vpnNetworkStackVariantManager.getVariant() == "ng"
```

Comment:
The toggle should be visible for the NG networking layer and hidden for the legacy one and enabled based on the value of `isEnabled`. The below lines are needed:
```
binding.vpnNewNetworkingLayerToggle.isVisible = vpnNetworkStackVariantManager.getVariant() == "ng"
binding.vpnNewNetworkingLayerToggle.isEnabled = isEnabled
```